source/theory.rst

```diff
@@ -440,16 +457,27 @@ The Morison Equation assumes that the fluid forces in an oscillating flow on a s
 
 where :math:`v` is the fluid particle velocity due to wave and current, :math:`C_{a}` is the coefficient of added mass, and :math:`\forall` is the displaced volume. 
 
+.. Note:: 
+	Adam:
+	What 'resulting force' can be approdimated by the Morison formulation? Some more information is needed here. My understanding is that it accounts for the undisturbed wave (Froude-Krylov force), viscous drag and added mass. So then should F_exc = F_rad = F_b = F_v = 0 when F_me is used? The below note only states that buoyance is neglected, not all of these components. (My understanding may be wrong of course, but if so I think that it shows some more description is needed.)
+
 .. Note:: 
 	WEC-Sim  does not consider buoyancy effects when calculating the forces from Morison elements. 
 
 For more information about application of Morison Elements in WEC-Sim, refer to :ref:`morison` section.
 
 Generalized Body Modes 
 ------------------------------------
-Additional generalized body modes (GBM) are inlcuded to account for solving multibody system with relative body motions or the the dynamics and structural deformation, assuming the modal properties are given or can be simply obtained in closed-form expressions or finite element analysis. Once the hydrodynamic coefficients that include these additional flexible DOF are obtained from the BEM solver, the 6DOF rigid body motion for each body and the additional GBM DOFs are solved together in one system of equations. 
+Additional generalized body modes (GBM) are included to account for solving a multibody system with relative body motions, dynamics, or structural deformation. This implementation assumes the modal properties are given, obtainable in closed-form expressions or with finite element analysis. Once the hydrodynamic coefficients that include these additional flexible DOF are obtained from the BEM solver, the 6DOF rigid body motion for each body and the additional GBM DOFs are solved together in one system of equations. 
+.. Note:: 
+	Adam:
+	This section is very short. Add some examples of GBM in WECs? What might it be used for?
```

Comment:
Please submit revisions via a PR, and I believe this is an application case for GBM.